velox/exec/tests/HashJoinTest.cpp

```diff
@@ -8243,4 +8243,184 @@ TEST_F(HashJoinTest, nanKeys) {
        makeFlatVector<int64_t>({1, 2})});
   facebook::velox::test::assertEqualVectors(expected, result);
 }
+
+DEBUG_ONLY_TEST_F(HashJoinTest, spillOnBlockedProbe) {
+  class BlockedNode : public core::PlanNode {
```

Comment:
Can we put this into OperatorTestUtil.h which should be useful in other cases? Thanks!